texar/torch/run/executor.py

```diff
@@ -1365,6 +1347,10 @@ def _try_get_data_size(executor: 'Executor'):
             self.write_log("Training terminated", mode='info')
         finally:
             self._train_tracker.stop()
+
+        # close the log files
+        self._close_files()
```

Comment:
Let's move this after end-of-training event is fired. It's possible for users to call `write_log` in those actions.

Reply:
Done in e175fa6